requirements.txt

```diff
@@ -1,18 +1,72 @@
-flake8
-ipdb
-ipython
-pytest
-pytest-cov
-Sphinx
-sphinxcontrib-applehelp
-sphinxcontrib-devhelp
-sphinxcontrib-htmlhelp
-sphinxcontrib-jsmath
-sphinxcontrib-qthelp
-sphinxcontrib-serializinghtml
-sphinx_rtd_theme
-versioneer
-cx-oracle
-pyodbc
-psycopg2-binary
-twine
+alabaster==0.7.12
```

Comment:
Strip version numbers as we want to be as flexible as possible.  It is probably easier just to add `ibm-db` and `pytest-sugar` to the existing list.